devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -28,6 +28,7 @@ metadata {
 		command "childOff", ["string"]
 
 		fingerprint profileId: "0104", inClusters: "0000, 0005, 0004, 0006", outClusters: "0000", manufacturer: "ORVIBO", model: "074b3ffba5a045b7afd94c47079dd553", deviceJoinName: "Switch 1"
+		fingerprint profileId: "0104", inClusters: "0006, 0005, 0004, 0000, 0003, 0B04, 0008", outClusters: "0019", manufacturer: "Aurora", model: "DoubleSocket50AU", deviceJoinName: "KK Aurora Smart Double Socket 1"
```

Comment:
You should take out the 'KK' from here and the name above since you're not testing anymore.

Reply:
You are right (it was committed by mistake).